assets/js/wfparameters/ColumnSelector.js

```diff
@@ -7,13 +7,17 @@ export default class ColumnSelector extends React.PureComponent {
   static propTypes = {
     name: PropTypes.string.isRequired,
     isReadOnly: PropTypes.bool.isRequired,
+    initialValue: PropTypes.string.isRequired,
+    onChange: PropTypes.func.isRequired,
```

Comment:
With any `func`, please add the signature in a comment. In this case: ` // onChange('A,B') => undefined`